features/step_definitions/webshop_steps.rb

```diff
@@ -0,0 +1,18 @@
+Given("the following product exist") do |table|
+  table.hashes.each do |product|
+  create(:product, product)
+  end
+end
+
+When("I am on the {string} page") do |string|
```

Comment:
This step is an assertion but the implementation performs an action. Also, you probably already have a similar step defined so reuse it if that's the case

Reply:
Will check it out